activerecord/test/cases/adapters/postgresql/postgresql_adapter_test.rb

```diff
@@ -123,6 +123,21 @@ def test_exec_insert_with_returning_disabled_and_no_sequence_name_given
         assert_equal expect.to_i, result.rows.first.first
       end
 
+      def test_exec_insert_default_values_with_returning_disabled_and_no_sequence_name_given
+        connection = connection_without_insert_returning
+        result = connection.exec_insert("insert into postgresql_partitioned_table_parent DEFAULT VALUES", nil, [], 'id')
+        expect = connection.query('select max(id) from postgresql_partitioned_table_parent').first.first
+        assert_equal expect.to_i, result.rows.first.first
+      end    
+
+      def test_exec_insert_default_values_quoted_schema_with_returning_disabled_and_no_sequence_name_given
+        connection = connection_without_insert_returning
+        result = connection.exec_insert('insert into "public"."postgresql_partitioned_table_parent" DEFAULT VALUES', nil, [], 'id')
+        expect = connection.query('select max(id) from postgresql_partitioned_table_parent').first.first
+        assert_equal expect.to_i, result.rows.first.first
+      end      
+
+
```

Comment:
Please be more prudent about whitespace issues like this (your `end` lines had trailing whitespace as well). We have to merge manually from the command line to fix issues like this, which takes a lot more time and means we can process fewer issues each day.